app/editor/src/features/admin/ingests/configurations/Newspaper.tsx

```diff
@@ -26,6 +27,54 @@ export const Newspaper: React.FC = (props) => {
   const sources = lookups.sources.map((s) => new OptionItem(s.name, s.code, s.isEnabled));
   const source = sources.find((t) => t.value === values.configuration.defaultSource);
 
+  const initialItems = () => {
+    const data: any[] = [];
+    let id = 0;
+    const keyValues = values.configuration.sources.split('&');
+    keyValues.forEach((x: string) => {
+      if (x !== '=') data.push({ id: ++id, name: x.split('=')[0], source: x.split('=')[1] });
```

Comment:
Will these array accessors fail if a user doesn't enter data correctly?

Reply:
It seems no so far based on the testing here, thanks